.coveragerc

```diff
@@ -13,10 +23,4 @@ exclude_lines =
     ## Don't complain if non-runnable code isn't run:
     ^[ ]*@unittest.skip\b
     ^[ ]{4}unittest.main()
-    if __name__ == .__main__.:
-
-
-[run]
-omit =
-    ## exclude debug.py from codecov report
-    */tests/debug.py
```

Comment:
we may want to add this omit to above maybe? i.e., add the debug.py to files omitted by coverage.